kafka-connect/kafka-connect-runtime/main/LICENSE

```diff
@@ -0,0 +1,1442 @@
+Project URL (from POM): https://github.com/census-instrumentation/opencensus-java
+License (from POM): The Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.perfmark  Name: perfmark-api  Version: 0.27.0
+Project URL (from POM): https://github.com/perfmark/perfmark
+License (from POM): Apache 2.0 - https://opensource.org/licenses/Apache-2.0
+
+--------------------------------------------------------------------------------
+
+Group: io.projectreactor  Name: reactor-core  Version: 3.4.41
+Project URL (from POM): https://github.com/reactor/reactor-core
+License (from POM): Apache License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.projectreactor.netty  Name: reactor-netty-core  Version: 1.0.48
+Project URL (from POM): https://github.com/reactor/reactor-netty
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.projectreactor.netty  Name: reactor-netty-http  Version: 1.0.48
+Project URL (from POM): https://github.com/reactor/reactor-netty
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: jakarta.activation  Name: jakarta.activation-api  Version: 1.2.1
+Project URL (from manifest): https://www.eclipse.org
+License (from POM): EDL 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
+License (from POM): Eclipse Public License v. 2.0 - https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+License (from POM): GNU General Public License, version 2 with the GNU Classpath Exception - https://www.gnu.org/software/classpath/license.html
+
+--------------------------------------------------------------------------------
+
+Group: javax.annotation  Name: javax.annotation-api  Version: 1.3.2
+Project URL (from manifest): https://javaee.github.io/glassfish
+Project URL (from POM): http://jcp.org/en/jsr/detail?id=250
+License (from POM): CDDL + GPLv2 with classpath exception - https://github.com/javaee/javax.annotation/blob/master/LICENSE
+
+--------------------------------------------------------------------------------
+
+Group: javax.servlet  Name: javax.servlet-api  Version: 3.1.0
+Project URL (from manifest): https://glassfish.dev.java.net
+Project URL (from POM): http://servlet-spec.java.net
+License (from POM): CDDL + GPLv2 with classpath exception - https://glassfish.dev.java.net/nonav/public/CDDL+GPL.html
+
+--------------------------------------------------------------------------------
+
+Group: javax.servlet.jsp  Name: jsp-api  Version: 2.1
```

Comment:
Should we still point to the old repository? https://github.com/javaee/javaee-jsp-api

Reply:
Done